README.md

```diff
@@ -202,6 +205,41 @@ DOJO_DOCKER_OPTIONS="--privileged"
 
 This method is suitable for running Dojo in Dojo. Dojo e2e tests use this method to test various Docker commands in a clean, separate Docker container. This way, such tests cannot affect Docker containers or images on Docker host (they have a safe environment to run in).
 
+### Public alpine image
+
+You can try any public Docker image. Example with an alpine image:
+```
+$ nano Dojofile
+$ cat Dojofile
+DOJO_DOCKER_IMAGE="alpine:3.16"
+$ dojo
+# now we run interactively in the Dojo Docker container
+/ # whoami
+root
+/ # pwd
```

Comment:
would be good if this example showcased the /dojo mounted directories as well. dojo/work and identity

Reply:
Also done :)